### PR TITLE
Add references to additional figures

### PR DESCRIPTION
**Pull request by rando2**
### Description of the proposed additions or changes

This addresses the concerns in #673 about adding in some of the awesome figures that had not been given a location in the manuscript.
~I'm going to mark it as a draft pending merge of #600 because that should fix the last broken figure issue we have.~
The changes to #600 were approved by the original author so if someone else is able to approve, we can merge.

### Related issues

#673 

### Checklist

- [ ] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

**Comment by rando2** [comment]
@agitter, something isn't working with the reference to the static figures in the images folder within content. I tried to follow the formatting we used for the other summary figure, so I'm not quite sure what's going wrong!

`**Summary of the relationships among topics covered in this review.**{#fig:overview secno=1}`

File: content/20.treatments.md [hunk 1]
```diff
@@ -818,11 +818,17 @@ There are over 100 vaccine candidates against SARS-CoV-2 in clinical trials.
...
-Diverse technology platforms include DNA, RNA, virus-like particle, recombinant protein, both replicating and non-replicating viral vectors, live attenuated virus, and inactivated virus approaches.
+Diverse technology platforms include DNA, RNA, virus-like particle, recombinant protein, both replicating and non-replicating viral vectors, live attenuated virus, and inactivated virus approaches (Figure @fig:vaccines).
 Given the wide range of vaccines under development, it is possible that some vaccine products may eventually be shown to be more effective in certain subpopulations, such as children, pregnant women, immunocompromised patients, the elderly, etc.
 The requirements for a successful vaccine trial and deployment are complex and may require coordination between government, industry, academia, and philanthropic entities [@doi:10.1126/science.abc5312].
 While little is currently known about immunity to SARS-CoV-2, vaccine development typically tests for serum neutralizing activity, as this has been established as a biomarker for adaptive immunity in other respiratory illnesses [@doi:10.1056/NEJMoa2022483].
 
+ ![
```
Comment (agitter) [inline content/20.treatments.md:826]:
```suggestion
![
```

File: content/20.treatments.md [hunk 2]
```diff
@@ -937,11 +943,16 @@ While trained immunity is an interesting possible avenue to complement vaccine d
 Currently, there are several possible prophylactics and therapeutics under investigation.
 Early public attention focused on potential treatments whose therapeutic/prophylactic value has subsequently been disproved or remains under question, such as HCQ and some nutraceuticals, respectively.
 Advances in prophylactics continue to develop, with several vaccines currently in clinical trials.
-As research has progressed, several potential approaches to treatment have emerged.
+As research has progressed, several potential approaches to treatment have emerged (Figure @fig:therapeutics).
 Most notably, remdesivir has been approved by the FDA for the treatment of COVID-19, and dexamethasone, which was approved by the FDA in 1958, has been found to improve outcomes for patients with severe COVID-19.
 As more evidence becomes available, the potential for existing and novel therapies to improve outcomes for COVID-19 patients will become better understood.
 At present, a number of promising avenues continue to be explored.
 
+**Mechanism of Action for Potential Therapeutics**
```
Comment (agitter) [inline content/20.treatments.md:951]:
```suggestion
![
**Mechanism of Action for Potential Therapeutics**
```

**Review by agitter — COMMENTED** [review]
Thanks for adding these.  I'm going to tweak the therapeutics figure to see if that fixes the build issue.

An upstream bug has been fixed so we may be able to get rid of `secno=1` in the figures, but let's not mess with that right now.

**Review by agitter — APPROVED** [review]
The figures appeared correctly in the PDF build.

**Pull request merged**